src/Analyzers/CSharp/Analyzers/xlf/CSharpAnalyzersResources.ko.xlf

```diff
@@ -274,7 +274,7 @@
       </trans-unit>
       <trans-unit id="typeof_can_be_converted_ to_nameof">
         <source>'typeof' can be converted  to 'nameof'</source>
-        <target state="new">'typeof' can be converted  to 'nameof'</target>
+        <target state="translated">'typeof'를 'nameof'로 변환할 수 있습니다.</target>
```

Comment:
Whatever the language is, it's weird not to have a space after `'typeof'` and `'nameof'`.

Reply:
Space is not common for   Korean and Japanese.